app/apollo/resolvers/cluster.js

```diff
@@ -286,28 +300,42 @@ const clusterResolvers = {
 
         checkComplexity( queryFields );
 
-        await validAuth(me, org_id, ACTIONS.READ, TYPES.CLUSTER, queryName, context);
+        let allAllowed = false;
+        try {
+          logger.info({req_id, user, org_id}, `${queryName} validating`);
```

Comment:
`validating` log message should be right after `try {` (could replace the 'enter' debug log line) so it 'includes' all the validation, e.g. checkComplexity.